includes/admin/import/class-batch-import-downloads.php

```diff
@@ -106,6 +106,21 @@ public function process_step() {
 					}
 				}
 
+				// Format the date properly
+				if ( ! empty( $args['post_date'] ) ) {
+
+					$timestamp = strtotime( $args['post_date'] );
```

Comment:
I'll add it, but i've verified on many timezones that adding this makes no difference in the output:
```
<?php
$timestamp = strtotime( '6/5/2016' );
var_dump($timestamp);
$timestamp2 = strtotime( '6/5/2016', current_time( 'timestamp' ) );
var_dump($timestamp2);

$date = date_i18n( 'Y-n-d H:i:s', $timestamp );
$date2 = date_i18n( 'Y-n-d H:i:s', $timestamp2 );
var_dump($date, $date2);
```
```
/app/public/wp-content/plugins/debug-bar-console/class-debug-bar-console.php(84) : eval()'d code:3:int 1465084800
/app/public/wp-content/plugins/debug-bar-console/class-debug-bar-console.php(84) : eval()'d code:5:int 1465084800
/app/public/wp-content/plugins/debug-bar-console/class-debug-bar-console.php(84) : eval()'d code:9:string '2016-6-05 00:00:00' (length=18)
/app/public/wp-content/plugins/debug-bar-console/class-debug-bar-console.php(84) : eval()'d code:9:string '2016-6-05 00:00:00' (length=18)
```

I did this with timezones at, before and after GMT.